simply/actor.py

```diff
@@ -147,12 +180,15 @@ def generate_order(self):
         # TODO replace order type by enum
         new = Order(np.sign(energy), self.t, self.id, self.cluster, abs(energy), price)
         self.orders.append(new)
+        return new
+
+    def next_time_step(self):
         # update schedule for next time step
+        # not part of generate_next_order, since order generation should not lead to next time step
+        # other things have to happen, for example matching, and supply of energy through the market
```

Comment:
If a battery exists, a supply of energy can happen, right? 
Do you mean inside the callback function `receive_market_results `, because this function might be called mutliple times i think